docs/source/conf.py

```diff
@@ -478,7 +543,29 @@ def autodoc_process_docstring(
                 ),
             )
 
-    # 2) Get the file names & line numbers
+    # 2) Insert "Returned in" admonition into classes that are returned from the Bot methods
+    try:
+        class_name = name.split(".")[-2]
+    except IndexError:
+        class_name = ""
+
+    # check for "__init__" ensures we generate "Returned in" admonition only once per class
+    if (
+        class_name in RETURN_ADMONITION_FOR_CLASS_NAME
+        and name.endswith("__init__")
+        and "slot wrapper" not in str(obj)  # filter out some side effects of __init__ check
+    ):
```

Comment:
can't we just check if `.. admonition:: Returned in` is already somewhere in `lines` 🤔

Reply:
Then it tries to insert the admonition e.g. in a docstring of `de_json()` and fails

Reply:
I see. the `what` argument probably also doesn't help?

Reply:
no, the `what` is "method" for `de_json()` as well, so I can't see a way to tell them apart

Reply:
> no, the `what` is "method" for `de_json()` as well, so I can't see a way to tell them apart

but shouldn't we be checking for "class"?

Reply:
Done